packages/financial-templates-lib/src/clients/FinancialContractEventClient.js

```diff
@@ -1,35 +1,35 @@
 // A thick client for getting information about an ExpiringMultiParty events. This client is kept separate from the
-// ExpiringMultiPartyClient to keep a clear separation of concerns and to limit the overhead from querying the chain.
+// FinancialContractEventClient to keep a clear separation of concerns and to limit the overhead from querying the chain.
 
-class ExpiringMultiPartyEventClient {
+class FinancialContractEventClient {
   /**
-   * @notice Constructs new ExpiringMultiPartyClient.
+   * @notice Constructs new FinancialContractEventClient.
    * @param {Object} logger Winston module used to send logs.
-   * @param {Object} empAbi Expiring Multi Party truffle ABI object to create a contract instance.
+   * @param {Object} financialContractAbi Expiring Multi Party truffle ABI object to create a contract instance.
    * @param {Object} web3 Web3 provider from truffle instance.
-   * @param {String} empAddress Ethereum address of the EMP contract deployed on the current network.
+   * @param {String} financialContractAddress Ethereum address of the Financial Contract contract deployed on the current network.
    * @param {Integer} startingBlockNumber Offset block number to index events from.
    * @param {Integer} endingBlockNumber Termination block number to index events until. If not defined runs to `latest`.
    * @return None or throws an Error.
    */
   constructor(
     logger,
-    empAbi,
+    financialContractAbi,
     web3,
-    empAddress,
+    financialContractAddress,
     startingBlockNumber = 0,
     endingBlockNumber = null,
-    contractType = "ExpiringMultiParty", // Default to EMP for now to enable backwards compatibility with other bots. This will be removed as soon as the other bots have been updated to work with these contract types.
+    contractType = "ExpiringMultiParty", // Default to Financial Contract for now to enable backwards compatibility with other bots. This will be removed as soon as the other bots have been updated to work with these contract types.
```

Comment:
comment might need to say defaults to expiring multi party

Reply:
done!